my_calc.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 
-x_len = 5
+x_len = int(input('How many arrays of 100? '))  # Not the best error handling!
 y_len = 100
```

Comment:
Do you think I should have allowed this param to be changed as well? 🤔

Reply:
I don't think this is needed yet.